src/chi-vue/src/views/CheckboxView.vue

```diff
@@ -1,31 +1,21 @@
 <template>
-  <div>
-    <h2>Checkbox</h2>
-    <h3>Base</h3>
-    <Checkbox @chiChange="(ev) => chiChange(ev)" id="example__checkbox_base" name="example__checkbox_base" />
-    <h3>Label</h3>
-    <Checkbox id="example__checkbox_label" label="Checkbox label" />
-    <h3>Selected: true</h3>
-    <Checkbox id="example__checkbox_selected" :selected="true" />
-    <h3>Selected: indeterminate</h3>
-    <Checkbox id="example__checkbox_indeterminate" :selected="'indeterminate'" />
-    <h3>Disabled</h3>
-    <Checkbox id="example__checkbox_disabled" :disabled="true" />
-  </div>
+  <h2>Checkbox</h2>
+  <h3>Base</h3>
+  <Checkbox @chiChange="(ev) => chiChange(ev)" id="example__checkbox_base" name="example__checkbox_base" />
+  <h3>Label</h3>
+  <Checkbox id="example__checkbox_label" label="Checkbox label" />
+  <h3>Selected: true</h3>
+  <Checkbox id="example__checkbox_selected" :selected="true" />
+  <h3>Selected: indeterminate</h3>
+  <Checkbox id="example__checkbox_indeterminate" :selected="'indeterminate'" />
```

Comment:
For non dynamic props we can pass the value directly:  `selected="indeterminate"`